### PR TITLE
Remove placeholder pages, tidy main page

### PR DESCRIPTION
**Pull request by lornajane**
Removed the pages that don't need to be there before launch. Also try to do something more useful with the landing page.

### CODE REVIEW

**Comment by lornajane** [comment]
This HowTo page is a great improvement. Can it work on the other table of contents pages too, like the Concepts pages? https://deploy-preview-52--developer-aiven-io-preview.netlify.app/docs/products/postgresql/concepts.html There might also be other places where the table of contents is used.

File: docs/tools/index.rst [hunk 1]
```diff
@@ -1,12 +1,13 @@
-Tools for Aiven
-===============
+Aiven Tools
```
Comment (ftisiot) [inline docs/tools/index.rst:1]: This is fine for the beta, however I would add some "squares" like the home page to make options more visible

Comment (ftisiot) [reply docs/tools/index.rst:1]: One square for each tool

File: index.rst [hunk 2]
```diff
@@ -1,70 +1,87 @@
...
+Explore
+-------
 
-Relational databases that you know and love ❤️
+Learn about the Aiven platform
 
 .. panels::
+    :card: shadow
+
+    |icon-postgres| **PostgreSQL** Powerful relational database platform. We have the latest versions, and all the extensions you need.
```
Comment (ftisiot) [inline index.rst:28]: "all extensions you need" <- isn't this a bit over-promising?

File: index.rst [hunk 3]
```diff
@@ -1,70 +1,87 @@
...
 
-    |icon-kafka| **Apache Kafka** For data streaming services, look no further than Apache Kafka.
+    ---
+
+    |icon-kafka| **Apache Kafka** Streaming data pipelines for the modern enterprise. Apache MirrorMaker2 and Kafka Connect also available.
 
     ---
 
-    |icon-kafka-connect| **Kafka Connect** To connect all the things into (source) and out of (sink) Apache Kafka.
+    |icon-elasticsearch| **ElasticSearch** Document database with specialist search features, bring your freeform ddocuments, logs or metrics, and make sense of them here.
```
Comment (ftisiot) [inline index.rst:54]: ddocuments <- bug

File: index.rst [hunk 4]
```diff
@@ -1,70 +1,87 @@
...
 
     ---
 
-    |icon-kafka-mirrormaker| **MirrorMaker2** Everything from one Apache Kafka to another.
+    |icon-cassandra| **Cassandra** High performance storage solution for large data quantities. This specialist data solution is a partitioned row store.
```
Comment (ftisiot) [inline index.rst:58]: "partitioned row store" I believe could be confusing? should we add NoSQL somewhere?

**Review by ftisiot — CHANGES_REQUESTED** [review]
Good shape, minor bugs to solve and I would add the various Aiven tools as squares like we do in the main home page for products

File: index.rst (hunk 4 above)
Comment (lornajane) [reply index.rst:58]: it isn't NoSQL, and its own website describes it that way. I thought it was a columnar database but apparently not!

File: index.rst (hunk 3 above)
Comment (lornajane) [reply index.rst:54]: good spot

File: index.rst (hunk 2 above)
Comment (lornajane) [reply index.rst:28]: I liked it! But you are probably correct, I'll tone it down.

File: docs/tools/index.rst (hunk 1 above)
Comment (lornajane) [reply docs/tools/index.rst:1]: I like this idea but I'm reluctant to delay this PR for it. Let's add it separately!

**Review by ftisiot — APPROVED** [review]
All good now

**Pull request merged**